src/Root.tsx

```diff
@@ -0,0 +1,25 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import PageNotFound from './components/PageNotFound';
+import { PeoplePage } from './components/PeoplePage';
+import HomePage from './components/HomePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
You can use ENUM for all paths and then you can use it in all places in your app.